<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiGridWidget</class>
 <widget class="QWidget" name="MultiGridWidget">
  <property name="windowTitle">
   <string>Multi grid method (orbit flattening)</string>
  </property>
  <layout class="QGridLayout" columnstretch="1,1">
   <item row="0" column="0" colspan="2">
    <widget class="ConfigSelect" name="configSelect"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox">
     <property name="title">
      <string>Monitor readouts:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="MonitorTable" name="monitorTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox">
     <property name="title">
      <string>Target values:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TargetTable" name="targetsTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox">
     <property name="title">
      <string>Calculated steerer corrections</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="FitSettingsWidget" name="fitSettingsWidget"/>
      </item>
      <item>
       <widget class="ResultTable" name="resultsTable">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="fitButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="prevButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="nextButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MonitorTable</class>
   <extends>QTableView</extends>
   <header>madgui/widget/correct/core.h</header>
  </customwidget>
  <customwidget>
   <class>TargetTable</class>
   <extends>QTableView</extends>
   <header>madgui/widget/correct/core.h</header>
  </customwidget>
  <customwidget>
   <class>ResultTable</class>
   <extends>QTableView</extends>
   <header>madgui/widget/correct/core.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigSelect</class>
   <extends>QWidget</extends>
   <header>madgui/widget/correct/configselect.h</header>
  </customwidget>
  <customwidget>
   <class>FitSettingsWidget</class>
   <extends>QWidget</extends>
   <header>madgui/widget/correct/fitsettings.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
